libs/dml/src/datamodel.rs

```diff
@@ -118,13 +118,27 @@ impl Datamodel {
             .expect("We assume an internally valid datamodel before mutating.")
     }
 
+    /// Finds a model by schema and name and returns a mutable reference.
+    pub fn find_model_schema_mut(&mut self, name: &str, schema: &Option<String>) -> &mut Model {
```

Comment:
Shouldn't the name already be unique without qualifying it with the schema attribute, in a valid schema? If this is about working with half-valid dml instances in introspection, that makes sense and this comment can be ignored.

Reply:
If we go the route that we talked about this will be unique, correct. I'll remove the method, it's unused since I removed the other deduplication approach already.